classes/layer-generator.bbclass

```diff
@@ -59,6 +59,35 @@ do_create_rootfs_archive() {
     mv ${IMAGE_ROOTFS_TAR} ${LAYER_WORK_DIR}/${DIGEST}
 }
 
+python do_create_whiteouts() {
+    import os
+
+    whiteoutPrefix = ".wh."
+    whiteoutOpaqueDir = whiteoutPrefix + whiteoutPrefix + ".opq"
```

Comment:
You mask whiteoutOpaqueDir by whiteoutPrefix. If only whiteoutPrefix has to be changed then whiteoutOpaqueDir will be changed implicitly.

Reply:
Done